content/overview/getting-started/_index.en.md

```diff
@@ -1,112 +1,148 @@
+
+Go back to the web UI and click **Define Link**. The web UI remembered the public keys of the actors and providers running, so you just need to pick them out of a dropdown selector at the top of the form, as shown below. For the 'Values' field, you can select a port number for the HTTP Server listener, and enter `port=8080` if you want to use port 8080 on localhost. (there should be no spaces before or after the equals sign). If you want the HTTP listener to use a different ip address, use "address" instead of port, and provide both IP address and the port, as in `address=10.1.1.1:9000`.
+
+![dashboard4](./washboard4.png)
 
-Once you see that the link has been advertised, you are ready to send a request to your actor.
+Once you see that the link has been added to the dashboard UI (you can also use `wash` to query this from the lattice), you are ready to send a request to your actor.
 
 #### Interacting with your actor
-In another terminal window (not the REPL), run the following command:
-```shell
+In another terminal window, run the following command:
+```
 curl localhost:8080/echo
 ```
 In response, you should receive your request object (notice the path argument):
+
 ```shell
-{"method":"GET","path":"/echo","query_string":"","headers":{"host":"localhost:8080","user-agent":"curl/7.64.1","accept":"*/*"},"body":[]}
+{"method":"GET","path":"/echo","query_string":"","headers":{},"body":[]}
 ```
+
 Feel free to try out different methods of making a request to your actor, including adding headers or using a different HTTP method to see different outputs.
 
-Instead of using `curl`, you can also _directly invoke_ actors' registered functions using `ctl call`. As mentioned before, the function that "echoes" this HTTP request is registered with the operation `HandleRequest`, and we can make this request directly to the actor if we supply the correct parameters.
+Instead of using `curl`, you can also _directly invoke_ actors' registered functions using `wash call`. The function that "echoes" this HTTP request is a part of the interface `wasmcloud:httpserver` and has the operation name `HandleRequest`. We can make this request directly to the actor if we supply the correct parameters.
+
+Here's an example of using `wash call` to mimic the previous `curl` command. Note that this is not interacting with the `HTTP Server` provider, and we don't need it to be running or linked for this operation to succeed:
 
-Here's an example of using `call` in the REPL to mimic our above `curl`. Note that this is not interacting with the `HTTP Server` provider, and we don't need it to be running or linked for this operation to succeed:
 ```shell
-ctl call MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5 HandleRequest {"method": "GET", "path": "/echo", "body": "", "queryString":"","header":{}}
+wash call MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5 HandleRequest '{"method": "GET", "path": "/echo", "body": "", "queryString":"","header":{}}'

```

Comment:
This does not work for me with echo `0.3.1` and wash `0.6.2` from @stevelr 's update rpc branch, I get back:
```
Error invoking actor: deserialization: deserialization: invalid type: unit value, expected byte array
```

Reply:
Aha... this might be because of the log that I get:
```
[error] Invocation failed anti-forgery validation check: Issuer of this invocation is not among the list of valid issuers
```

@autodidaddict we'll have to add `wash` as a valid issuer, or otherwise pass the cluster key won't we?